### PR TITLE
[OAP]Update Conda recipes for 0.9.0 release.

### PR DESCRIPTION
**Pull request by yao531441**
What changes were proposed in this pull request?

Update Conda recipes for 0.9.0 release.

### CODE REVIEW

File: docs/OAP-Installation-Guide.md [hunk 1]
```diff
@@ -38,7 +38,7 @@ Dependencies below are required by OAP and all of them are included in OAP Conda
 
 ```bash
 conda activate oapenv
-conda install -c intel -c conda-forge -y oap=0.9.0
+conda install -c conda-forge -c intel -y oap=0.9.0
 ```
 Once finished steps above, you have completed OAP dependencies installation and OAP building, and will find built OAP jars in `/root/miniconda2/envs/oapenv/oap_jars/`
```
Comment (zhixingheyi-tian) [inline docs/OAP-Installation-Guide.md:44]: For "Conda Requirements", Line: 25,  please separate the Conda Installation and usages

Thanks

**Comment by zhixingheyi-tian** [comment]
@HongW2019 

Please have a review

File: dev/scripts/prepare_oap_env.sh [hunk 2]
```diff
@@ -147,7 +147,7 @@ function prepare_memkind() {
   fi
   cd memkind/
   git pull
-  git checkout v1.10.1-rc2
+  git checkout v1.10.1
```
Comment (HongW2019) [inline dev/scripts/prepare_oap_env.sh:150]: need change to rc2.

**Comment by HongW2019** [comment]
LGTM

**Pull request merged**